e2e/operator/operator_test.go

```diff
@@ -0,0 +1,22 @@
+package operator
+
+import (
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+
+	"github.com/openshift/cluster-kube-apiserver-operator/e2e"
+)
+
+func TestOpenShiftNamespace(t *testing.T) {
```

Comment:
We need to establish framework that will make sure the operator is initially in the state and version we expect since the test reuse the same cluster and are disruptive.

Reply:
We can let this PR in first so the CI is setup and we have better way to debug setups like this.